solution.rb

```diff
@@ -0,0 +1,29 @@
+TEXT = File.read('speech.txt')
+
+def find_words text
+  words_regex = /\w+[a-z']*\w+/
```

Comment:
I think this would make a little more sense defined as a constant outside the method.